skcosmo/preprocessing/flexible_scaler.py

```diff
@@ -114,15 +114,9 @@ class KernelFlexibleCenterer(TransformerMixin, BaseEstimator):
     but with additional parameters, relative to which centering
     is carried out:
 
-
-    :param ref_cmean: an array with means for each column.
-    :type ref_cmean: array of shape (1, n_features)
-
-    :param ref_mean: an average for the whole kernel matrix
-    :type ref_mean: array
```

Comment:
Why just delete a piece of documentation? These are important parameters, and you just remove them.

Reply:
They're parameters for the _fit_ function, not the overall class. This isn't an appropriate place to have them.